languages/en_GB/poll.json

```diff
@@ -32,10 +33,13 @@
   "warning.redactor": "You're using Redactor. Do <strong>not</strong> edit the markup manually. Instead, re-open the poll creator.",
 
   "vote": "Vote",
+  "update_vote": "Update vote",
+  "remove_vote": "Remove vote",
   "to_voting": "To Voting",
   "to_results": "Results",
   "voting_unavailable_title": "Voting unavailable",
   "voting_unavailable_message": "This poll has ended or has been marked as deleted. You can still view the results.",
+  "voting_update_disallowed_message": "You already voted and voting update has been disallowed for this poll",
```

Comment:
Suggest: "You have already voted and changing your vote is not allowed for this poll"